server/odc-service/src/main/java/com/oceanbase/odc/service/collaboration/environment/EnvironmentService.java

```diff
@@ -143,17 +141,6 @@ private Environment innerDetailWithoutPermissionCheck(@NonNull Long id) {
     }
 
     private Environment entityToModel(@NonNull EnvironmentEntity entity) {
-        Environment environment = environmentMapper.entityToModel(entity);
```

Comment:
Simply remove the unnecessary query as we do not need the creator/modifier information for now.